### PR TITLE
Add CycleOps Fluid2 power calculator

### PR DESCRIPTION
**Pull request by oldnapalm**

### CODE REVIEW

**Comment by oldnapalm** [comment]
Sorry, I made some changes later that should not be in the pull request, my intention was to pull only commits a1d4aa4 and 20d2102

File: CycleOpsFluid2PowerCalculator2.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from AbstractPowerCalculator import AbstractPowerCalculator
```
Comment (dhague) [inline CycleOpsFluid2PowerCalculator2.py:1]: This file, ending in "2", seems to be surplus to requirements

**Review by dhague — CHANGES_REQUESTED** [review]

File: CycleOpsFluid2PowerCalculator2.py (hunk 1 above)
Comment (oldnapalm) [reply CycleOpsFluid2PowerCalculator2.py:1]: Yes, that file is not needed, it's another power curve, I put it into my fork just to store it. My intention was to pull only commits a1d4aa4 and 20d2102, I didn't know changes I made later were going to be added do pull request. Can you select only those commits or do you want me to start over? Sorry, I'm new to github.

Comment (dhague) [reply CycleOpsFluid2PowerCalculator2.py:1]: Unfortunately I can only approve the entire pull request. Maybe create a new branch at your last commit, and then reset the original branch to the correct commit and push that to the PR branch?

Comment (dhague) [reply CycleOpsFluid2PowerCalculator2.py:1]: If that's all too complicated, just close this PR, create a new branch at the correct commit, and push that as a new PR.

**Pull request closed without merging**